README.md

```diff
@@ -1823,7 +1823,7 @@ Other Style Guides
 
     ```javascript
     // good
-    (function (global) {
+    (function(global) {
```

Comment:
actually this space is required. There must always be a space after the `function` keyword, as described [here](https://github.com/airbnb/javascript#functions--signature-spacing)